docs/sphinx/index.rst

```diff
@@ -65,22 +65,22 @@ Example Usage
     from datetime import datetime
     from elasticsearch import Elasticsearch
 
-    es = Elasticsearch("http://localhost:9200")
+    client = Elasticsearch(cloud_id="YOUR_CLOUD_ID", api_key="YOUR_API_KEY")
```

Comment:
I don't want to encourage Cloud IDs which don't really have an advantage over simple endpoints:

```suggestion
    client = Elasticsearch("http://localhost:9200/", api_key="YOUR_API_KEY")
```

Reply:
added those three and made the change you suggested, thanks!